### ISSUE TITLE
AsymmetricDepolarizingChannel overrides function num_qubits with a property

### ISSUE THREAD

**Issue by Strilanc** [quantumlib/Cirq#4185]
```
# works
n = cirq.DepolarizingChannel(0.1).num_qubits()

# throws
n = cirq.AsymmetricDepolarizingChannel(p_x=0, p_y=0.1, p_z=0).num_qubits()
```

**Comment by balopat** [comment]
This should be easy to fix: just remove the @property decorator from the num_qubits() method. This is technically going to be a breaking change so whoever takes this issue should mark the PR with the "breaking change" label and add a release note to the description.

**Comment by dabacon** [comment]
Fixed in #4652
